pkg/db/db.go

```diff
@@ -0,0 +1,153 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/content-services/content-sources-backend/pkg/config"
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	pg "gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+// GetUrl Get database config and return url
+func GetUrl() string {
+
+	conf := config.Get()
+
+	return fmt.Sprintf(
+		"user=%s password=%s dbname=%s host=%s port=%d sslmode=disable",
+		conf.DBUser,
+		conf.DBPassword,
+		conf.DBName,
+		conf.DBHost,
+		conf.DBPort,
+	)
+}
+
+// Connect returns pointer to a new gorm DB connection
+func Connect() (*gorm.DB, error) {
+	dbURL := GetUrl()
+	DB, err := gorm.Open(pg.Open(dbURL), &gorm.Config{})
```

Comment:
this actaully creates a local DB variable due to the :=  and doesn't set the global one (it stays nil)

Reply:
right, the global DB is there by mistake